### DISCUSSION TITLE
Problem z dostępem do Synology

### DISCUSSION THREAD

**Discussion by Qu3aX** [AzureCommunityPL/group#320] [Pomoc]
Witam,

Mam problem po podłączeniu komputera do domeny azurowej przestaje działać komunikacja z serwerem synology oraz innym linuxie, który udostępnia pliki. Wygląda na to, że azur blokuje protokół smb (pingi przechodzą, udostępnione pliki na innym serwerze na Windowsie normalnie są widoczne). Niestety nigdzie nie mogę znaleźć żadnej informacji czy i w jakim miejscu mogłoby być to zablokowane. Licencje, które mam to Microsoft 365 Business Premium.

**Comment by smereczynski** [comment]
Ale o jaką usługę Azure chodzi? Co ma do tego M365? Opisz proszę nieco bardziej składnie co starasz się zrobić i co nie działa.

**Reply by Qu3aX** [reply]
Mamy laptopy, które nie są wpięte do domeny azurowej, możemy z nich bez problemu dostać się do serwerów linuxowych oraz Windows serwera, na których są udostępnione zasoby sieciowe. Po wpięciu do domeny azurowej nie można połączyć się z żadnym z nich. Po zalogowaniu się na lokalnego admina, który nie jest w domenie Azurowej znów wszystko działa.

**Reply by smereczynski** [reply]
@eklime to chyba jakieś tam kerberosy w domenie? Nie jestem Windowsowy, więc wolę wywołać mądrzejszych.

**Reply by Qu3aX** [reply]
Próbowałem również teraz z laptopem, który jest w całkowicie nowej domenie azurowej z konfiguracją domyślną i problem jest identyczny tzn. Logując się przez .\Admin na komputer mogę dostać się do udostępnionych zasobów, ale już logując się kontem azurowym nie mam dostępu do niczego.

**Reply by eklime** [reply]
Pomimo wielu użytych znaków, nadal niewiele wiadomo z czym dokładnie masz problem.
Dostajesz jakieś błędy? Na zasoby publiczne też nie masz dostępu?

**Reply by Qu3aX** [reply]
Łącząc się do udostępnionych zasobów sieciowych, do których można dostać się z dowolnymi poświadczeniami, będąc w domenie azurowej nie wyskakuje okienko do wpisania poświadczeń tylko od razu pojawia się ten błąd:

Błąd sieciowy 
System Windows nie może uzyskać dostępu do \\192.168.1.12 (i 3 innych urządzeń, które nie są ze sobą w żaden sposób połączone)
Nie masz uprawnień dostępu do \\192.168.1.12. Skontaktuj się z administratorem sieci, aby uzyskać dostęp.

Będąc zalogowany do tego samego komputera przez .\Admin już to okienko już nam wyskakuje.

**Reply by TomaszOledzki** [reply]
Na pewno masz problem z uwierzytelnianiem. Na pewno potrzebujesz informatyka do rozwiązania tego problemu.

Zacząłbym od analizy logów np. Security na kliencie i loga w którym samba zapisuje log uwierzytelniania (bo to jakiś Linux hostuje pliki, tak?).

**Reply by smajchrz** [reply]
Po wpisaniu następującego hasła w Google otrzymasz cała stronę wyników i możliwych rozwiązań.

**azure ad join cannot connect to smb share**

Czy środowisku jest lokalna domena? Czy może zasoby z tych serwerów są udostępniane użytkownikom stworzonym na tych serwerach a ich nazwy logowania są identyczne jak te używane na laptopach?
